reference/conanfile/methods/run.rst

```diff
@@ -0,0 +1,19 @@
+.. _reference_conanfile_methods_run:
+
+run()
+=====
+
+Running commands
+----------------
+
+.. code-block:: python
+
+    run(self, command, stdout=None, cwd=None, ignore_errors=False, env="", quiet=False, shell=True, scope="build")
+
+
+``self.run()`` is a helper to run system commands and throw exceptions when errors occur,
```

Comment:
It has a very important responsibility: it is the one injecting the calls to activate environment scripts.